### PR TITLE
AO3-6549 Remove ignore of auto collection invite column

### PR DESCRIPTION
**Pull request by brianjaustin**
# Pull Request Checklist

* [X] Have you read ["How to write the perfect pull request"](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)?
* [X] Have you read the [contributing guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
* [X] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
* [X] Have you added the [Jira](https://otwarchive.atlassian.net) issue number
  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
* [X] Do you have fewer than 5 pull requests already open? If not, please wait
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6549

## Purpose

Remove the ignore for the column that was deleted in #4505

## Testing Instructions

Reference Jira

## References

Are there other relevant issues/pull requests/mailing list discussions?

## Credit

Brian Austin (they/he)

### CODE REVIEW

**Comment by brianjaustin** [comment]
Since the change was so small, I also went through the RuboCop issues for that file (and added tests). They should be addressed eventually either way, but if we prefer not to do all this at once I can revert some or all of them

**Review by ceithir — COMMENTED** [review]
I fear you've opened Pandora's box: Now you have to update a regexp to a new syntax, and so must add a bunch more tests to make quite sure nothing was broken in the process.

What's currently in the PR seems good though. :+1:

**Comment by brianjaustin** [comment]
@ceithir thanks for the review! Based on https://stackoverflow.com/a/3632051, I think it won't actually be too bad (just the extra test which would fail with the old regex)

File: spec/models/preference_spec.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+      context "when the user's preference has skins enabled" do
+        before do
+          user.preference.update!(disable_work_skins: false)
+        end
+
+        it "returns false" do
+          expect(Preference.disable_work_skin?("foo")).to be(false)
+        end
+      end
```
Comment (ceithir) [inline spec/models/preference_spec.rb:40]: Nitpick: Splitting between context and test might be a bit overkill here:
```suggestion
      it "returns false when the user's preference has skins enabled" do
        user.preference.update!(disable_work_skins: false)
        expect(Preference.disable_work_skin?("foo")).to be(false)
      end
```

File: spec/models/preference_spec.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+      context "when the user's preference has skins disabled" do
+        before do
+          user.preference.update!(disable_work_skins: true)
+        end
+
+        it "returns true" do
+          expect(Preference.disable_work_skin?("foo")).to be(true)
+        end
+      end
```
Comment (ceithir) [inline spec/models/preference_spec.rb:50]:
```suggestion
      it "returns true when the user's preference has skins disabled" do
        user.preference.update!(disable_work_skins: true)
        expect(Preference.disable_work_skin?("foo")).to be(true)
      end
```

**Review by ceithir — APPROVED** [review]

**Pull request merged**